.pylintrc

```diff
@@ -0,0 +1,85 @@
+[MASTER]
+
+extension-pkg-whitelist= 
+
+ignore=CVS,migrations
+
+disable=print-statement,
```

Comment:
Do we really need all these disabled? Just pick the specific ones that we can afford to ignore.